source/common/network/address_impl.cc

```diff
@@ -128,8 +128,8 @@ Ipv4Instance::Ipv4Instance(const std::string& address, uint32_t port) : Instance
   }
 
   friendly_name_ = absl::StrCat(address, ":", port);
-  validateIpv4Supported(friendly_name_);
   ip_.friendly_address_ = address;
+  validateIpv4Supported(friendly_name_);
```

Comment:
nit: this doesn't matter functionally, but can you move this up two lines so there's no diff here at all? Less noise to think through in the change that way.

Reply:
Oops. Thanks for catching this.